docs/ios_device_check.md

```diff
@@ -1,81 +1,158 @@
-# Configuring iOS DeviceCheck
+# Exposure Notification Server
 
-To verify device integrity, Apple provides a service called [DeviceCheck][dc].
-This service is most commonly used to store per-developer bits, but it can also
-be used to verify the legitimacy of an iOS device.
+## Verifying iOS devices using DeviceCheck
 
-As part of the DeviceCheck process, the server receives a `deviceToken` from a
-verified iOS device. This token is generated by a client application see the
-[DeviceCheck docs][dc] for information on how to generate this token in your
-app. Note that the token is generated on the _client_. The _server_ does not
-generate this token, but it does validate it's authenticity.
+Apple uses the [DeviceCheck API](https://developer.apple.com/documentation/devicecheck)
+to verify device integrity. As part of the DeviceCheck verification process, the
+server receives a token that is generated on the device to verify the iOS
+device.
 
-When the server receives the `deviceToken`, it needs to communicate with Apple's
-servers to verify the authenticity of the token. This request requires
-authentication and authorization. Authentication is provided via a signed JWT
-and authorization is provided based on the grants given to the signing key. The
-server automatically constructs and signs these JWTs, but you must provide the
-signing keys.
+When the server receives the `device_token` from an iOS device, it needs to
+communicate with Apple's servers to verify the authenticity of the token. This
+request requires authentication and authorization. Authentication is provided
+via a signed JSON Web Token (JWT) and authorization is provided based on the
+grants given to the signing key. You must provide the signing keys that the
+server uses to construct and sign the JWT.
 
-Each _team_ requires a different key for verification. If you are running a
-multi-tenant server that supports iOS apps from multiple developers, you will
-need to acquire and register DeviceCheck for _each_ team.
+Each developer, called a "team", requires a different key for verification. If
+you are running a multi-tenant server that supports iOS apps from multiple
+developers, you will need to acquire and register DeviceCheck for each team.
 
+## Getting a Team ID
 
-## Getting the Team ID
+**A Team ID requires a valid and active Apple iOS Developer subscription.**
 
-The DeviceCheck API requires you to specify your Team ID. **This requires a
-valid and active iOS Apple developer subscription.**
-
-1.  Visit https://developer.apple.com/account/#/membership and sign in to your
-    developer account.
+To get a Team ID:
```

Comment:
This is incorrect language. They are not provisioning a new team.

Reply:
Reworded.